hack/make-rules/verify.sh

```diff
@@ -109,13 +120,19 @@ function run-checks {
   for t in $(ls ${pattern})
   do
     local check_name="$(basename "${t}")"
-    if is-excluded "${t}" ; then
-      echo "Skipping ${check_name}"
-      continue
-    fi
-    if ${QUICK} && ! is-quick "${t}" ; then
-      echo "Skipping ${check_name} in quick mode"
-      continue
+    if [[ ! -z ${VERIFY_CHECKS} ]]; then
```

Comment:
make this `${VERIFY_CHECKS:-}` since we set nounset

Reply:
Done.